src/Humanizer/FrequencyHumanizeExtensions.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Humanizer
+{
+    public static class FrequencyHumanizeExtensions
+    {
+        public static string Humanize(this Frequency input)
+        {
+            string output = "every";
+            switch (input.OccurrenceFrequency)
+            {
+                case FrequencyEnum.Never:
+                    output = "never";
```

Comment:
Ok thank you, I've amended the other things as you told me to do, now I'll
try to find out how to do this last one, then I'll do another pull request.
 Le 30 mars 2015 00:04, "Alexander Zaytsev" <notifications@github.com> a
écrit :

> In src/Humanizer/FrequencyHumanizeExtensions.cs
> <https://github.com/MehdiK/Humanizer/pull/400#discussion_r27359748>:
>
> > +﻿using System;
> > +using System.Collections.Generic;
> > +using System.Linq;
> > +using System.Text;
> > +
> > +namespace Humanizer
> > +{
> > +    public static class FrequencyHumanizeExtensions
> > +    {
> > +        public static string Humanize(this Frequency input)
> > +        {
> > +            string output = "every";
> > +            switch (input.OccurrenceFrequency)
> > +            {
> > +                case FrequencyEnum.Never:
> > +                    output = "never";
>
> I mean, to put these strings to .NET resources, so we can get translations
> out of the box
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/MehdiK/Humanizer/pull/400/files#r27359748>.
>